src/errors/IsolatedCircuitError.ts

```diff
@@ -0,0 +1,11 @@
+import { BrokenCircuitError } from './BrokenCircuitError';
+
+/**
+ * Exception thrown from {@link CircuitBreakerPolicy.execute} when the
+ * circuit breaker is open.
+ */
+export class IsolatedCircuitError extends BrokenCircuitError {
+  constructor() {
+    super(`Execution prevented because the circuit breaker is open`);
```

Comment:
Can change to `public message = '...'`, avoids ctor.

Reply:
Avoiding the ctor means it doesn't get put in the stacktrace though, which is probably undesirable